README.md

```diff
@@ -160,6 +161,9 @@ plugins:
 **Note!**
 Once you modify your `nomad.yaml` file adding `include`, all the default plugins will be disconnected, so you will need to include them as well.
 
+## How to cite this work
```

Comment:
@JosePizarro3 like this?

Reply:
Pizarro, J.M., Boydas, E.B., Daelman, N., Ladines, A.N., Mohr, B. & Rudzinski, J.F., NOMAD Simulations [Computer software]. https://doi.org/xxxxx

Reply:
tho I think a new option will open on the menu on the right for citation, if i see nomad-material-processing

Reply:
ah ok, thanks. Well, it does't hurt to have it in the README, it's a bit more visible...merge?